lib/Web2Print/Processor/HeadlessChrome.php

```diff
@@ -38,6 +38,7 @@ protected function buildPdf(Document\PrintAbstract $document, $config)
         $web2printConfig = Config::getWeb2PrintConfig();
         $web2printConfig = $web2printConfig->get('headlessChromeSettings');
         $web2printConfig = json_decode($web2printConfig, true);
+        $web2printConfig = array_merge($web2printConfig, ($config->headlessChromeSettings ?? []));
```

Comment:
could you please also update arguments [here ](https://github.com/pimcore/pimcore/blob/723d46a2be247f81e4776cec90b104796dbe30c6/lib/Event/DocumentEvents.php#L121)and also add headless chrome example on how to extend headless chrome settings in the comments? thanks!